addons/mail/models/mail_canned_response.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class MailCannedResponse(models.Model):
+    """
+    Canned Response: content that will automatically replace the shortcut of your choosing. This content can still be adapted before sending your message.
+    """
+
+    _name = "mail.canned_response"
```

Comment:
Don't use "_" in name, we always use dots to distinguish from table name notably (yes I know, ir.config_parameter, the author should be killed)
```suggestion
    _name = "mail.canned.response"
```